### PR TITLE
Use pangeo/pangeo-notebook as base image

### LINKED ISSUE #19: add h5netcdf to farallon hub env
getting error when trying to write a zarr file. tried to pip install, but maybe need it in image. thanks.

### LINKED ISSUE #22: Duplicate launcher entries
On the staging hubs, where the pangeo/pangeo-notebook is used as the base image as per https://github.com/2i2c-org/pangeo-hubs/pull/15, some of the launcher entries show up two times, but the environment is the same:

duplicates

We should probably find a way to get rid of one of these.

### PR DESCRIPTION
**Pull request by yuvipanda**
Curated upstream image! But it doesn't have R, so we
add it. Our environment.yml file only adds packages that
aren't in the base image.

Ref #3
Closes https://github.com/2i2c-org/pangeo-hubs/issues/19
Closes https://github.com/2i2c-org/pangeo-hubs/issues/22

### CODE REVIEW

**Comment by choldgraf** [comment]
🤩

chrome_JsVTH4gtKb

File: deployments/ohw/image/Dockerfile [hunk 1]
```diff
@@ -104,9 +63,6 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 
 USER ${NB_USER}
 
-# FIXME: Split this into two files
-# We can then install things that require python / jupyter separate
-# from scientific python libs - speeding up image build times a lot
 COPY environment.yml /tmp/
 
 RUN conda env update -p ${CONDA_DIR} -f /tmp/environment.yml
```
Comment (yuvipanda) [inline deployments/ohw/image/Dockerfile]:
```suggestion
RUN conda env update --name ${CONDA_ENV} -f /tmp/environment.yml
```

The base PANGEO image's root conda environment is called 'notebook', and available via the 'CONDA_ENV' environment variable. This is what the notebook server is started from, so our notebook related packages (like jupyter-server-proxy or jupyter-rsession-proxy) need to be installed there. I think this is also where the python kernels start by default, so all our scientific python packages also need to be there.

File: deployments/ohw/image/Dockerfile [hunk 2]
```diff
@@ -104,26 +63,23 @@ COPY rsession.conf /etc/rstudio/rsession.conf
...
-RUN conda env update -p ${CONDA_DIR} -f /tmp/environment.yml
+RUN conda env update --name ${CONDA_ENV} -f /tmp/environment.yml
 
 COPY install-jupyter-extensions.bash /tmp/install-jupyter-extensions.bash
 RUN /tmp/install-jupyter-extensions.bash
 
 # Install IRKernel
-RUN r -e "install.packages('IRkernel', version='1.1.1')" && \
-    r -e "IRkernel::installspec(prefix='${CONDA_DIR}')"
+RUN conda run -n ${CONDA_ENV} r -e "install.packages('IRkernel', version='1.1.1')" && \
```
Comment (yuvipanda) [inline deployments/ohw/image/Dockerfile]: Add a line of comment documenting why `conda run` was needed here?

**Comment by choldgraf** [comment]
Is there anything holding this one back? Checks seem green, no comments on things to resolve?

**Comment by GeorgianaElena** [comment]
Hey @choldgraf! This is already deployed on the staging hubs. We just need to wait a little so that folks test it a bit to make sure everything works before deploying it to prod.

**Comment by yuvipanda** [comment]
@choldgraf waiting for some responses in https://gitter.im/pangeo-data/FI-hub

**Comment by choldgraf** [comment]
Ah right - do the folks that should be testing it know that they need to test it out?

**Comment by GeorgianaElena** [comment]
Yep! It's on the Gitter chat.

**Comment by GeorgianaElena** [comment]
Deployed on prod :tada:

**Comment by rabernat** [comment]
Fantastic, thanks so much for this integration!

Going forward, 2i2c folks should feel free to engage on https://github.com/pangeo-data/pangeo-docker-images/ when it comes to updating this image. Many of the changes here could be pushed upstream.

In the long term, we need to find a way to make updates to the pangeo environments less dependent on support from the admins. This image has a lot of churn, since we always need to be on the bleeding edge of the many package releases. I notice we are pinning to a particular commit, but the pangeo-notebook image is often updated several times a week. So this will be stale in a few weeks.

**Pull request merged**